src/darwin/Framework/CHIP/templates/CHIPCommandPayloadsObjc-src.zapt

```diff
@@ -18,6 +18,13 @@ NS_ASSUME_NONNULL_BEGIN
   }
   return self;
 }
+
+- (NSString *)description
+{
+  NSString *descriptionString = [NSString stringWithFormat:@"<CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params: {{#zcl_command_arguments}}_{{asStructPropertyName label}}: %@; {{/zcl_command_arguments}}>"{{#zcl_command_arguments}},{{#if isArray}}_{{asStructPropertyName label}}{{else if (isOctetString type)}}[_{{asStructPropertyName label}} base64EncodedStringWithOptions:0]{{else}}_{{asStructPropertyName label}}{{/if}}{{/zcl_command_arguments}}];
```

Comment:
Also, does it make sense to use `NSStringFromClass([self class])` for the type name, so we don't have to duplicate the exact structure of it?

Reply:
Yes, this make sense.

Reply:
Updated